server/src/lib/user.ts

```diff
@@ -9,24 +9,29 @@ export class User {
   public playerId: number;
   public murderer: boolean;
   public realPlayer: boolean;
-  public location: Hallway;
+  public currentLocation: Hallway;
 
   constructor(name: string, id: number, realPlayer: boolean) {
     this.name = name;
     this.playerId = id;
     this.murderer = false;
     this.realPlayer = realPlayer;
-    this.location = null;
+    this.currentLocation = null;
   }
 
   public isMurderer(): boolean {
     return this.murderer;
   }
 
+  public location(): Hallway {
+    return this.currentLocation;
```

Comment:
User.currentLocation is a public class parameter, we don't need a separate getter. User.currentLocation vs User.location()